pkg/config/controller_config.go

```diff
@@ -39,7 +42,8 @@ func (cfg *ControllerConfig) BindFlags(fs *pflag.FlagSet) {
 		"Set the controller log file - if not specified logs are written to stdout")
 	fs.IntVar(&cfg.MaxConcurrentReconciles, flagMaxConcurrentReconciles, defaultMaxConcurrentReconciles, ""+
 		"Maximum number of concurrent reconcile loops")
-	fs.BoolVar(&cfg.EnableCloudWatchLogs, flagEnableCloudWatchLogs, false, "If enabled, policy decision logs will be streamed to CloudWatch")
+	fs.BoolVar(&cfg.EnablePolicyEventLogs, flagEnablePolicyEventLogs, false, "If enabled, policy decision logs will be collected & logged")
```

Comment:
Flow specific logs will be rather useful for debugging. Since this is an initial release, will be helpful to have it turned on by default for the first few releases. It should only log an entry per flow and not per packet (for accepted flows). Obviously, this change allows an user to turn it off if it becomes too verbose for them..

Reply:
The problem with turning it on by default is not the amount of logs being produced, but the fact the the busy loop which is collecting the decision logs is consuming a full CPU core on every node. From a users point of view this unexpected and undesired default behavior.

Reply:
Sorry for the delay. We had some internal discussions on what the default should be for this flag and we agreed on disabling this by default considering the agent runs as a daemonset. So, we should be good with the current PR. We will look to enable it by default, once we find a way to optimize the CPU usage. Thanks for the PR.